spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -127,48 +125,48 @@
         let(:item) { FactoryBot.create(:v2_tailoring, :with_tailored_values, policy: parent, os_minor_version: 8) }
 
         it 'returns tailored values and default set of rules, but unselected' do
-          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
-
-          tailoring_file = Nokogiri::XML(response.body).remove_namespaces!
-          selected, deselected = tailoring_file.xpath('//Profile/select')
-                                               .partition { |sel| sel.attributes['selected'].value == 'true' }
-          values = tailoring_file.xpath('//Profile/set-value/@idref')
+          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: format)
 
           expect(response).to have_http_status :ok
-          expect(selected).to be_empty
-          expect(deselected).not_to be_empty
-          expect(deselected.map { |r| r.attributes['selected'].value }.uniq).to eq(['false'])
+          expect(rules).not_to be_empty
+          expect(selected_rules).to be_empty
           expect(values).not_to be_empty
         end
       end
 
       context 'with randomly distributed rules' do
+        let(:canonical_profile) do
+          FactoryBot.create(
+            :v2_profile,
+            rule_count: 5,
+            value_count: 5,
+            security_guide: FactoryBot.create(:v2_security_guide, rule_count: 3, os_major_version: 9),
+            os_major_version: 9,
+            ref_id_suffix: 'bar',
+            supports_minors: [8]
+          )
+        end
```

Comment:
Why was this reintroduced?

Reply:
To ensure the tailoring file has some added tests.

Reply:
In that case you might not need the `for_tailoring` trait.